### ISSUE TITLE
As a learner, I can't login with a non- @gov.bc.ca address

### ISSUE THREAD

**Issue by allanhaggett** [bcgov/moodle#101]
As per a MyHR CRM ticket that's been forwarded to me (not sure if I should paste the issue # here; erring on cautious by not), we have a learner with an Learner.Name@**wcat.bc.ca** email address who cannot access PSA Moodle. 

We also recently had a Liquor Distribution Board (LDB) person who couldn't access, though that's a lot more understandable LDB people don't have IDIRs. That person was also a facilitator, not a learner (see below).

The @wcat.bc.ca has an IDIR and can login to the PSA Learning System (PSALS or ELM).

Is this an AzureAD limitation? Will switching to Keycloak make it go away? 

If we need to start manually creating accounts for learners, how do we obtain that persons bcgovGUID to use? We already create manual accounts for select teachers so that we can have external facilitators, but those folx don't require updated learning records (they can't access PSALS anyhow), so we don't really need to worry about their GUIDs.

**Comment by warrenchristian1telus** [comment]
@allanhaggett Maybe we could have one (or more) of them test the KeyCloak login in dev now? For me it worked with Azure, but not KeyCloak. If other groups will have issues with KeyCloak, it would probably be best to sort it out before we switch, if at all possible. Not sure if this would get into the issue of new KeyCloak Realms (which I believe we still cannot get created at this time).

If this is an issue in both KeyCloak and Azure, and especially KeyCloak, I think we may need to find some credentials (or user(s)) that we can access to validate any fixes in dev / test.

**Comment by allanhaggett** [comment]
We have started work to integrate BC Services Card SSO login which will officially cover this use case. We are good to delay this integration until capacity allows for a fulsome participation in the fairly lengthy process of setting it up. We can continue to use Moodle-native manual accounts to support non-gov-consultants or students who have IDIR access issues, but we need to be aware that this is not in compliance with the appropriate legislation in this area.

Closing this issue in favour of creating a new issue specific to BC Services Card.
